app/src/main/java/lt/ausra/note/notesActivity/NotesActivityViewModel.kt

```diff
@@ -19,4 +19,18 @@ class NotesActivityViewModel : ViewModel() {
         }
         _notesLiveData.value = NoteRepository.instance.notes
     }
+
+    fun filterNotesList(enteredText: String) {
```

Comment:
This probably should be done on the `viewModel` side, as I can see you're using those